addons/core/functions/fnc_radioReplaceProcess.sqf

```diff
@@ -40,5 +40,20 @@ if (GVAR(use_saved_lr_setting) && {isNil QGVAR(saved_active_lr_settings)}) then
 };
 
 if ((time - TF_respawnedAt > 5) and {alive TFAR_currentUnit}) then {
-    false call DFUNC(requestRadios);
+    private _countPrototypes = {
```

Comment:
I don't see why this should be a seperate function if you only call it once.

Reply:
I would break out, after I've found the first one. There is no need to check the other items.